sites/svelte-5-preview/src/routes/docs/content/03-appendix/02-breaking-changes.md

```diff
@@ -179,3 +179,7 @@ In Svelte 4, `null` and `undefined` were printed as the corresponding string. In
 ### `bind:files` values can only be `null`, `undefined` or `FileList`
 
 `bind:files` is now a two-way binding. As such, when setting a value, it needs to be either falsy (`null` or `undefined`) or of type `FileList`.
+
+### Bindings now react to form resets
+
+Previously, calling `form.reset()` wasn't notified by bindings, as such values could get out of sync with the DOM. Svelte 5 fixes this by placing a `reset` listener on the document and invoking bindings where necessary.
```

Comment:
> calling `form.reset()` wasn't notified by bindings

Sounds like the wrong way around, shouldn't that be something like "`form.reset()` did not notify bindings" or "did not update bindings"?

Reply:
Also, not sure if this is obvious enough to people, but the current mechanism relies on bubbling, which can of course be prevented via `stopPropagation` ([REPL](https://svelte-5-preview-jufubosx8-svelte.vercel.app/#H4sIAAAAAAAACqWPQWrDMBBFrzIMhSRg6r0qGXqBUpplkoUSj4vAHglpFFqM714U47hdZNWNQJqv99-M2LmeEqrDiGwHQoWvIWCF8h3KJV2pF8IKk8_xUl50ukQXpDnyUXoSYC978SFQCwaeklih7UboS-DAXiDNs9Nm97L8SA_if6K6XntYdz4Ot0btOGSBs-NWXW2fyYyrwAT1HDpnEc9QdjCREknzUU5dz4Om4Bckv62WCn7T7s3g-UYxI4FpgJ5L_D36YD-tOM_b3QSP7NJ_1PZ3rQWDFQ6-dZ2jFpXETNNp-gF8y2VZxAEAAA==)).

Reply:
Initially I used the capture phase but then switched to the bubble phase because I thought there could be false positives, but I'm now realizing that's moot, so I'm gonna switch back to the capture phase which won't have this problem